app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,64 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @questions = Question.all
+  end
+
+  def show
+    @question.visit_count += 1
+    @question.save
```

Comment:
write a method on `Question` to do this. @question.increment_view_count (this method could likely skip validations).